fastlane/Fastfile

```diff
@@ -715,6 +721,25 @@ end
       xcargs: { UI_TEST_ACCOUNT: options[:test_account] || '' }
     )
   end
+
+  #####################################################################################
+  # pick_test_account_and_run_ui_tests
+  # -----------------------------------------------------------------------------------
+  # Firstly tries to occupy a free test account, and then passes the account to 
+  # `run_ui_tests` lane. Frees up the account afterwards.
+  # -----------------------------------------------------------------------------------
+  # Usage:
+  # bundle exec fastlane pick_test_account_and_run_ui_tests [device:<Name of the iOS Simulator>]
+  #
+  # Example:
+  # bundle exec fastlane pick_test_account_and_run_ui_tests device:"iPhone 12"
+  #####################################################################################
+  desc "Occupy a free test account and run UI tests with it"
+  lane :pick_test_account_and_run_ui_tests do | options |
+    occupy_test_account
+    run_ui_tests(options.merge({ test_account: "simplenoteuitest-ios-#{$used_test_account_index+1}@a8c.com" }))
```

Comment:
Nitpick.

It might be a tiny bit simpler to avoid the index shifting logic. What do you think?

```suggestion
    run_ui_tests(options.merge({ test_account: "simplenoteuitest-ios-#{$used_test_account_index}@a8c.com" }))
```

Or is there a reason why we shouldn't have a `simplenoteuitest-ios-0@a8c.com` test account?

Reply:
Yes, of course! I did not think this through when asked JB to create 1-3 accounts. The zero-account should be ready today

Reply:
Account created already - so no index shift anymore! 🙇

Reply:
Cool. Thanks for that.